crates/api_crud/src/site/update.rs

```diff
@@ -208,3 +183,201 @@ impl PerformCrud for EditSite {
     Ok(res)
   }
 }
+
+fn validate_update_payload(
+  site_regex: Option<String>,
+  federation_enabled: bool,
+  private_instance: bool,
+  edit_site: &EditSite,
+) -> LemmyResult<()> {
```

Comment:
Do we need a check to make sure that the site does exist (opposite of the `local_site.site_setup` in create)? Not immediately clear to me if this would also work as a way to create the initial site.

Reply:
Not really necessary IMO, because it never sets that site_setup field in the form builder.